pkg/cstor/cstor.go

```diff
@@ -506,3 +499,35 @@ func (p *Plugin) SetVolumeID(unstructuredPV runtime.Unstructured, volumeID strin
 
 	return &unstructured.Unstructured{Object: res}, nil
 }
+
+func (p *Plugin) getVolInfo(volumeID, snapName string) (*Volume, error) {
+	// To support namespace re-mapping for cloud snapshot,
```

Comment:
ok.. got to see this comment.. so, still some work is pending for namespace re-mapping for cloud snapshots

Reply:
namespace re-mapping for cloud snapshot is out of scope for this PR. We will add support for this in near future.